GliaWidgets/Sources/ViewController/Call/CallViewController.swift

```diff
@@ -115,10 +115,11 @@ final class CallViewController: EngagementViewController {
                 view.isVisitrOnHold = isOnHold
             case .transferring:
                 view.setConnectState(.transferring, animated: true)
-            case .showSnackBarView(let text):
+            case .showSnackBarView:
```

Comment:
same to bind method

Reply:
Fixed 246e9175ed076c002b5ba21490d0622a73767154